sensor_array/analysis/brute_force_analysis.py

```diff
@@ -320,7 +329,7 @@ def create_bins(gases, num_bins, mof_list, element_pmf_results):
         # Jenna Bins
         # lower_lim = lower_comp
         # upper_lim = upper_comp + (upper_comp-lower_comp)/(num_bins)
-        bin_points = np.linspace(lower_lim, upper_lim, num=num_bins+1, endpoint=True)
+        bin_points.append(np.linspace(lower_lim, upper_lim, num=num_bins+1, endpoint=True))
```

Comment:
I need you to walk me through this commit ca6e013a5c11488af46f5a14a5dba656f585235b